components/clp-package-utils/clp_package_utils/scripts/native/archive_manager.py

```diff
@@ -387,8 +389,9 @@ def _delete_archives(
 
     logger.info(f"Finished deleting archives from the database.")
 
+    archive_output_dir: Path = archives_dir / dataset if dataset is not None else archives_dir
```

Comment:
This piece of code has been repeated at serveral places, but since it's simple I feel maybe we don't need a specific method for it. Any different opinion?